config.lua.dist

```diff
@@ -80,6 +80,7 @@ mysqlSock = ""
 -- may cause high CPU usage with many players and potentially affect performance!
 -- NOTE: forceMonsterTypesOnLoad loads all monster types on startup to validate them.
 -- You can disable it to save some memory if you don't see any errors at startup.
+-- When activating maintenance mode, only players with a position above tutors will be able to login to the server.
```

Comment:
This is actually already handled here https://github.com/otland/forgottenserver/blob/412950218c5e01062a5e2458d363802a5cea3115/src/protocolgame.cpp#L152-L154

You'd just have to give that flag to groups that you want to be able to log in.